packages/custom-url/src/Infrastructure/Repository/CustomUrlRepository.php

```diff
@@ -0,0 +1,81 @@
+<?php
+
+/*
+ * This file is part of Sulu.
+ *
+ * (c) Sulu GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\CustomUrl\Infrastructure\Repository;
```

Comment:
```suggestion
namespace Sulu\CustomUrl\Infrastructure\Doctrine\Repository;
```

doctrine namespace missing